pkg/descheduler/controllers/migration/evictor/interpreter.go

```diff
@@ -37,7 +38,7 @@ var (
 	ErrTooManyEvictions = errors.New("TooManyEvictions")
 )
 
-type FactoryFn func(client kubernetes.Interface) Interface
+type FactoryFn func(client kubernetes.Interface, policyGroupVersion string) Interface
```

Comment:
It is recommended to keep the parameters of this function type unchanged, and directly modify the implementation-aware eviction policyGroupVersion of NativeEvictor.

Reply:
Fine. Changed it. 

@eahydra